autoload/codeql.vim

```diff
@@ -148,7 +148,7 @@ function! codeql#uriToFname(uri) abort
         let l:orig_fname = v:lua.vim.uri_to_fname(a:uri)
     endif
 
-    if isdirectory(s:database.'/src')
+    if isdirectory(s:database.'/src') && stridx(l:orig_fname, '/') != 0
```

Comment:
Im not sure about this one, since in line 152 I was appending `orig_fname` to `/src` without any `/` in the middle. That makes me think that relative paths also start with `/` otherwise it would not have worked in the first place.

I think a different approach would be to replace the `isdirectory()` check with `filereadable(l:orig_fname)`, that way, if it exists it should return `orig_fname` and otherwise, it should prepend the db path

Reply:
Cool, makes sense. I will try to build a relative database next time and see what happens. Will update the PR then.